src/data/contributors/Kartikey.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Kartikey Dubey",
+  "skills": ["Java", "c", "C++","Python"],
+  "cause": "Learn and Explore new things",
+  "image": "Avatar7",
```

Comment:
Choose from the already created avatars. If you want to add a new Avatar then it should be a part of a new PR.

Reply:
okay